### ISSUE TITLE
Better icon for Draw Tool

### ISSUE THREAD

**Issue by morevnaproject** [synfig/synfig#776]
Currently icon for Draw Tool looks like this -
![screenshot_017](https://user-images.githubusercontent.com/332868/54493179-71865480-48ff-11e9-9c6c-7b7cd3e04bc3.png)

For many users this is not intuitive. They often choose Sketch Tool (which has completely different purpose) instead of Draw Tool.

So, I am looking for some icon, which better reflects the nature of Draw Tool. 

Details about Draw Tool - https://wiki.synfig.org/Doc:Creating_Shapes#Using_tablet_to_draw_shapes

**Comment by An0u** [comment]
![Splice_Tool_Synfig](https://user-images.githubusercontent.com/46160598/54497259-6677f800-48f0-11e9-9adb-90a408d7d54e.png)

How does something like this look as a draft ? Is there somewhere I post and ask for feedback? I have several sketches done and also some research.

**Comment by morevnaproject** [comment]
Yes, please post everything here - this is a place where we can collect feedback.

**Comment by corbeeresearch** [comment]
https://cdn.iconscout.com/icon/premium/png-256-thumb/draw-edit-pen-pencil-writing-33320.png

**Comment by morevnaproject** [comment]
@corbeeresearch I like the icon, but in this case Draw Tool will look exactly the same as Sketch Tool
![screenshot_008](https://user-images.githubusercontent.com/332868/55684726-ec7be180-5978-11e9-8f32-ae040b85d4e9.png)

Possibly, we can think about better icon for Sketch Tool as well?

**Comment by maitreyi0505** [comment]
Hey @morevnaproject ! So I designed icons for this issue, considering the difference between the 2 tools, please have a look:

Draw tool:
![image](https://user-images.githubusercontent.com/44134797/56088732-5e39bb00-5ea4-11e9-9dbe-951531808a3f.png)


Sketch tool:
![image](https://user-images.githubusercontent.com/44134797/56088723-43ffdd00-5ea4-11e9-8e9e-b4fdb90f9b80.png)

If you wish I can create a new one or make any changes in these if needed!

**Comment by morevnaproject** [comment]
@maitreyi0505 I like the icon for Draw Tool! But the difference with Sketch Tool can be hard to recognize hen icon is small. 

How about utilizing a concept of pen/pencil? Here is an icon from Tango project - 
![240px-Applications-office svg](https://user-images.githubusercontent.com/332868/56136620-5ae62280-5fbd-11e9-936f-f71b570bf2b1.png)

Another idea is to play with ink pen concept - https://thenounproject.com/term/ink-pen/75540/

**Comment by maitreyi0505** [comment]
Yes I was also skeptical about the similarity! So just confirming- Pencil for draw tool and ink-pen for sketch tool?

**Comment by b3u** [comment]
@morevnaproject trying to revive this issue: What is the difference between the pen and draw tools?

**Comment by ankit-kumar-dwivedi** [comment]
Hi, @b3u 


> What is the difference between the pen and draw tools?

I think by pen tool you are referring to [sketch tool](https://wiki.synfig.org/Sketch_Tool)
The sketch tool is used to freehand draw anything on the canvas, it is similar to the pencil tool or freehand draw in similar software while [draw tool](https://wiki.synfig.org/Draw_Tool) is more advanced it can be used to draw complex shapes using splines.


> Details about Draw Tool - https://wiki.synfig.org/Doc:Creating_Shapes#Using_tablet_to_draw_shape

**Comment by morevnaproject** [comment]
@Mark11110 And which should be in "storyboarding" section? ^__^
